projects/schematics/src/migrations/3_0/component-deprecations/data/close-account-modal.component.migration.ts

```diff
@@ -0,0 +1,14 @@
+import { CLOSE_ACCOUNT_MODAL_COMPONENT } from '../../../../shared/constants';
+import { ComponentData } from '../../../../shared/utils/file-utils';
+
+export const CLOSE_ACCOUNT_MODAL_COMPONENT_MIGRATION: ComponentData = {
+  // projects\storefrontlib\src\cms-components\myaccount\close-account\components\close-account-modal\close-account-modal.component.ts
+  selector: 'cx-close-account-modal',
+  componentClassName: CLOSE_ACCOUNT_MODAL_COMPONENT,
+  removedProperties: [
+    {
+      name: 'userToken',
+      comment: `'userToken' property has been removed.`,
```

Comment:
make sure to tell the user to look at isLoggedIn$ instead as it got replaced

also mention it in the 3.0.md

Reply:
Done